File: src/test/samples/sample.py
#region FirstRegion
x = 42
#endregion

# region SecondRegion
class MyClass:
    #  region InnerRegion
    def method(self):
        pass
    #          endregion

    #region
    def method2(self):
        pass
    # endregion

# endregion

File: src/test/samples/sample.clj
;#region FirstRegion
(def x 42)
;;#endregion

;;;;   #region SecondRegion  
(defn my-class []
    ;   #region    InnerRegion   
    (defn my-method [] nil)
    ;;   #endregion    ends InnerRegion  

    ;;   #region  
    (defn my-method2 [] nil)
    ;;#endregion  
)
;; #endregion

